### ISSUE TITLE
Button element is not recognized but <input type="button"/> is.

### ISSUE THREAD

**Issue by Jedimaster0** [cpriest/SnapLinksPlus#84]
I have an issue where buttons are not recognized by Snap Links Plus. The button is added to an element and if I change it to <input type="button"/> then it works fine. However the "<button></button>" element does not seem to be detected. Is this a legitimate issue or am I doing something wrong?

Here is my code:
$("#" + strCaseID + "_CASES_CASE_NUMBER").html('<a href=/' + strCaseID + '>' + strTicket + '</a>&nbsp;&nbsp;<button id="own' + i + '" class="own noContext sfo_button" type="button" value="' + strCaseID + '">' + strPwnEnabled + '</button>');

$('.own').mouseup(function(event) {
		var thisCaseID = $(this).val();
		switch (event.button) {
			case 1: // Middle Mouse Button
				self.port.emit("ownTicket", "https://c.na6.visual.force.com/apex/TakeOwnershipPage?scontrolCaching=1&id=" + thisCaseID + ",false,false");
				break;
			case 2: // Right Mouse Button
				self.port.emit("ownTicket", "https://c.na6.visual.force.com/apex/TakeOwnershipPage?scontrolCaching=1&id=" + thisCaseID + ",true,false");
				break;
			default:
				self.port.emit("ownTicket", "https://c.na6.visual.force.com/apex/TakeOwnershipPage?scontrolCaching=1&id=" + thisCaseID + "," + sfoOptions.tabBackground + "," + sfoOptions.tabAutoClose);
		}
		if (sfoOptions.hideTicket){ $("#sfo_row_" + thisCaseID).slideUp("fast", function(){ $(this).remove(); sfoOptions.ticketCount - 1; }); }
	});

**Comment by cpriest** [comment]
If I understand you correctly, you're referring to <input type="button"> elements, in which case that is not a supported use case.  Before I took over the project it only worked on <A> anchors, I got involved because I wanted it to work with checkboxes and radio buttons.  

While personally I do use it to open a single link in the background and I can see a similar use case to 'click' a button I wouldn't be adding that feature without the ability to control it. 

If I'm misunderstanding, let me know.  This is something that may be considered for a next major release which I'm considering will be a ground-up rewrite.
